recipes/pyjulia/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.4" %}
+{% set name = "julia" %}
+
+package:
+  name: pyjulia
```

Comment:
**Note to others:** The package is called `julia` on PyPI. Though that will conflict with the `julia` package (compiler/interpreter) in `conda-forge`, so using the repo name is a reasonable second choice. Hence it is called `pyjulia` here.

Reply:
> This seems fine to me once the issues above are addressed.

We're going to get a new version with the LISENCE.md file released on PyPI, and them I'll update that. 

@ViralBShah we can also rename on PyPI to pyjulia for consistency, thoughts ?

Reply:
Yes,  I think that makes sense for consistency.